src/main/java/frc/robot/Robot.java

```diff
@@ -105,6 +108,28 @@ public void teleopInit() {
    */
   @Override
   public void teleopPeriodic() {
+    String gameData;
+    gameData = DriverStation.getInstance().getGameSpecificMessage();
+    if (gameData.length() > 0) {
+      switch (gameData.charAt(0)) {
+      case 'B':
+        positionControlColor = 'B';
+        break;
+      case 'R':
+        positionControlColor = 'R';
+        break;
+      case 'Y':
+        positionControlColor = 'Y';
+        break;
+      case 'G':
+        positionControlColor = 'G';
+        break;
+      default:
+        break;
+      }
+    } else {
+
```

Comment:
If you're not going to do anything in the `else` statement, then why include it?
This will only ever run if we receive an empty string from the FMS (aka when there's no data available).

Reply:
It's going to be common to receive an empty or null string from FMS, so that does need to be handled gracefully.  But I agree that an empty else clause should be removed.